tests/tuner/test_lr_finder.py

```diff
@@ -283,3 +284,15 @@ def training_step_end(self, outputs):
     trainer = Trainer(default_root_dir=tmpdir)
     num_training = 3
     trainer.tuner.lr_find(model=model, num_training=num_training)
+
+
+def test_multiple_lr_find_calls_gives_same_results(tmpdir):
+    """Tests that lr_finder gives same results if called multiple times."""
+    dm = ClassifDataModule()
+    model = ClassificationModel()
+
+    trainer = Trainer(default_root_dir=tmpdir, max_epochs=2)
+    all_res = [trainer.tuner.lr_find(model, datamodule=dm).results for _ in range(3)]
+
+    for k in all_res[0].keys():
+        assert all(all_res[0][k] == curr_lr_finder[k] for curr_lr_finder in all_res[1:])
```

Comment:
```suggestion
    model = BoringModel()

    trainer = Trainer(default_root_dir=tmpdir, max_epochs=2)
    all_res = [trainer.tuner.lr_find(model).results for _ in range(3)]

    for k in all_res[0].keys():
        assert all(all_res[0][k] == curr_lr_finder[k] for curr_lr_finder in all_res[1:])
```

there is no need to use a complex model, we can use the boring model. 
test fails on master.

Reply:
sometimes `lr_find` fails when using `BoringModel` since it doesn't converge and it just logs the exception. So I prefer to use a model with lr_finder tests that at least converge to confirm that it works as expected.

Reply:
don't we want to set a seed then? I  mean just initially for the test